eslint.config.js

```diff
@@ -30,7 +31,7 @@ const eslintPluginTestsRecommendedConfig =
 
 export default [
 	{
-		ignores: ["**/tests/fixtures/", "**/dist/"],
+		ignores: ["**/tests/fixtures/", "**/dist/", "test.css"],
```

Comment:
Added this `test.css` for local testing purposes, similar to the [eslint repo](https://github.com/eslint/eslint/blob/52f5b7a0af48a2f143f0bccfd4e036025b08280d/eslint.config.js#L94)